server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/HttpJmapAuthentication.java

```diff
@@ -21,32 +21,54 @@
 
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.Optional;
+import java.util.concurrent.TimeUnit;
 
 import org.apache.http.client.ClientProtocolException;
 import org.apache.http.client.fluent.Request;
 import org.apache.http.client.fluent.Response;
 import org.apache.http.client.utils.URIBuilder;
 import org.apache.http.entity.ContentType;
 import org.apache.james.jmap.api.access.AccessToken;
+import org.hamcrest.Description;
+import org.hamcrest.TypeSafeMatcher;
 
+import com.jayway.awaitility.Awaitility;
+import com.jayway.awaitility.Duration;
+import com.jayway.awaitility.core.ConditionFactory;
 import com.jayway.jsonpath.JsonPath;
 
 public class HttpJmapAuthentication {
 
+    private static final ConditionFactory CALMLY_AWAIT = Awaitility.with()
+            .pollInterval(Duration.FIVE_HUNDRED_MILLISECONDS)
+            .and().with()
+            .pollDelay(Duration.FIVE_HUNDRED_MILLISECONDS)
```

Comment:
I don't notice a longer duration, and i prefere to wait 500ms more than having a fail after 25 minutes.